source/components/debugger/dbconvert.c

```diff
@@ -354,6 +354,8 @@ AcpiDbConvertToPackage (
 
     Elements = ACPI_ALLOCATE_ZEROED (
         DB_DEFAULT_PKG_ELEMENTS * sizeof (ACPI_OBJECT));
+	if (!Elements)
```

Comment:
i think the indentation is a bit too deep 😅

Reply:
Sorry about that, but I didn't make such an ugly indentation when modifying the code. I believe this is caused by a web page rendering issue.

Reply:
LGTM